_integration-schemas/klaviyo/global_exclusions.md

```diff
@@ -0,0 +1,32 @@
+---
+tap: "klaviyo"
+version: "1.0"
+
+name: "global_exclusions"
+doc-link: ""
+singer-schema: "https://github.com/singer-io/tap-klaviyo/blob/master/tap_klaviyo/schemas/global_exclusions.json"
+description: ""
+
+replication-method: "Full Table"
+
+api-method:
+    name: ""
+    doc-link: "https://www.klaviyo.com/docs/api/metrics"
+
+attributes:
+  - name: "email"
```

Comment:
This should be the table's primary key. I'm getting that from [here](https://github.com/singer-io/tap-klaviyo/blob/master/tap_klaviyo/__init__.py#L67), which isn't a typical implementation, but it's the closest thing I can find.